com.gitlab.davem.ClamTk.yaml

```diff
@@ -0,0 +1,117 @@
+app-id: com.gitlab.davem.ClamTk
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: clamtk
+rename-icon: clamtk
+copy-icon: true
+finish-args:
+  - '--filesystem=host'
```

Comment:
Does this app need to write something outside xdg dirs? If not, maybe set `host:ro` instead?

Reply:
True, fixed.

Reply:
Actually that was a bad idea, because ClamTk can quarantine scanned files. So yeah, it does need to write (or more particularly, delete) outside the XDG dirs.